SnapWar/ViewControllers/SWFullScreenImageViewController.swift

```diff
@@ -0,0 +1,49 @@
+
+import UIKit
+
+class SWFullscreenImageViewController: UIViewController {
+    var imageView: UIImageView
+    
+    required init(named: String) {
+        guard let image = UIImage(named: named) else {
+            fatalError("Image not found")
+        }
+        self.imageView = UIImageView(image: image)
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        view?.backgroundColor = SWColorType.black
+        setup()
+    }
+    
+    func setup() {
+        view.addSubview(imageView)
+
+        imageView.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
im thinking we should have a swimageview that does this automatically so we don't have to replicate this layout code everytime

Reply:
^ Will create this in a separate branch